app/Models/User.php

```diff
@@ -17,7 +17,9 @@ class User extends Authenticatable
      *
      * @var array<int, string>
      */
-
+    public function tasks(){
+        return $this->hasMany(Tasks::class, 'user_id');
+    }
```

Comment:
```suggestion
    public function tasks()
    {
        return $this->hasMany(Tasks::class, 'user_id');
    }
```

- Modal `User` cũng cần thêm relationship ngược lại nhé bạn
- Xem lại giúp mình các relationship trong laravel nha! Mình vẫn chưa thấy đủ những quan hệ cơ bản.

Reply:
Vâng để em xem lại ạ